asciidoc/courses/aura-dashboards/course.adoc

```diff
@@ -1,30 +1,37 @@
-= Building Dashboards in Neo4j Aura
+= Building Dashboards in Neo4j
 :categories: administrator:1, beginners:5, foundation:1, aura:1
 :status: draft
 :duration: 2 hours
-:caption: Learn about data visualisation in Neo4j Aura.
-:key-points: Aura dashboards, exploratory data analysis, data import and visualization, exploring relationships in data.
+:caption: Learn about data visualization using Neo4j Dashboards.
+:key-points: Neo4j dashboards, exploratory data analysis, data import and visualization, exploring relationships in data.
 
-In this course you will learn about Aura Dashboards, a feature for visualising and exploring graph data directly within the Aura environment.
-You will learn how to access Aura Dashboards, import and visualize data, create interactive dashboards using AI and Cypher queries, and share dashboards with others.
+In this course you will learn about Neo4j Dashboards, a feature for visualizing and exploring graph data directly within Neo4j.
+You will learn how to access Dashboards, import and visualize data, create interactive dashboards using AI and Cypher queries, and share dashboards with others.
 
-You will explore the Aura Dashboards interface, learn about different visualization options for data representation, and discover the built-in tools for data exploration and visualization.
+You will explore the Dashboards interface, learn about different visualization options for data representation, and discover the built-in tools for data exploration and visualization.
 
-This course will prepare you to visualize and understand the Neo4j databases in the cloud with confidence.
+This course will prepare you to visualize and understand Neo4j databases with confidence. While the examples use Neo4j Aura for ease of access, the same features are available in Neo4j Enterprise Edition on-premises.
 
 == Duration
 
 {duration}
 
 == Prerequisites
 
-Before starting this course, you should have a basic understanding of graph databases and basic knowledge on Neo4j. If you are new to Neo4j, you can familiarize yourself with the database with the link:/courses/neo4j-fundamentals/[Neo4j Fundamentals course^], the link:/courses/aura-fundamentals/[Aura Fundamentals course^] and the link:/courses/cypher-fundamentals/[Cypher Fundamentals course^].
+Before starting this course, you should have:
+
+* A basic understanding of graph databases and Neo4j
+* Access to a Neo4j database - either:
+  ** Neo4j Aura (free tier available) - recommended for new users
+  ** Neo4j Enterprise Edition on-premises with Dashboards enabled
```

Comment:
maybe link out to the docs where they can find how to enable dashboards on enterprise